src/main/java/ru/timeconqueror/tcneiadditions/nei/TCAspectStringifyHandler.java

```diff
@@ -0,0 +1,45 @@
+package ru.timeconqueror.tcneiadditions.nei;
+
+import codechicken.nei.api.IStackStringifyHandler;
+import com.djgiannuzz.thaumcraftneiplugin.ModItems;
+import com.djgiannuzz.thaumcraftneiplugin.items.ItemAspect;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraftforge.fluids.FluidStack;
+import thaumcraft.api.aspects.Aspect;
+import thaumcraft.api.aspects.AspectList;
+
+public class TCAspectStringifyHandler implements IStackStringifyHandler {
+    @Override
+    public NBTTagCompound convertItemStackToNBT(ItemStack stack, boolean saveStackSize) {
```

Comment:
please save Count to nbt if saveStackSize is true. see DefaultStackStringifyHandler

Reply:
![ezgif com-gif-maker (8)](https://user-images.githubusercontent.com/31038811/168127752-e29b6e87-663c-4832-9302-3023a5ed884e.gif)
Count needed to keep this behavior

In the future need save correct aspect amount

Reply:
Ah, I got it